### PR TITLE
update all.json

### PR DESCRIPTION
**Pull request by ical10**
|Site|Urlscan|
|--|--|
|blockfieldanalysis.net|https://urlscan.io/result/8a09fd85-4c53-46a6-8199-f24429e14796/|
|blockfieldsanalysis.net|https://urlscan.io/result/b10430e7-6cfd-4166-8232-3d00e0dcec64/|
|coin-claim.com|https://user-images.githubusercontent.com/14356826/171787304-5b13e53a-1a0e-4773-a398-438895aaea2c.png|
|colablond.cc|https://urlscan.io/result/5e6d6abc-2296-428e-9e4c-9d6f97618731/|
|collablawd.cc|https://urlscan.io/result/f8879284-b374-47e0-bd85-5f09c76cb1b9/|
|collaplanb.cc|https://urlscan.io/result/6134225d-c400-47f0-938f-7a8e07cd5a50/|
|cronos-assistnode.com|https://urlscan.io/result/053b2f9a-1bc4-465c-b1ed-0c000d01ba4d/|
|cronos-bscsystem.com|https://urlscan.io/result/91402441-04be-4102-bff0-012d8bc9f843/|
|cronos-bscupdate.com|https://urlscan.io/result/d8e986b4-1bd2-458c-bf0d-fa9171aadad7/|
|cronos-ecosystem.com|https://urlscan.io/result/0e828d90-843b-4809-b6d2-eeca914aea0a/|
|cronos-scandapp.com|https://urlscan.io/result/6bd5770e-a4d3-43ed-8887-9d0a11c7b89f/|
|cronos-scandesk.com|https://urlscan.io/result/63e821ea-8233-4ca1-9b5e-5da5e9d3a0b7/|
|cronos-scansystem.com|https://urlscan.io/result/49688e4c-e16d-4845-b7e1-1f58d1090d88/|
|cronos-system.com|https://urlscan.io/result/b81165e3-e92a-47cd-abf4-c4d4adc5afa8/|
|ragavio.ro|https://urlscan.io/result/3b929aff-be1b-48d7-9c5b-c14b4cdadba5/|
|safenode-launchpad.co|https://urlscan.io/result/ee516a1d-9ba6-4c42-91c7-65682beba89c/|
|secured.all-in-oneconnect.com|https://urlscan.io/result/79623232-7936-43f1-b2c4-e6b27552fd6a/|
|syncnodelist.com|https://user-images.githubusercontent.com/14356826/171787898-d6e4f25f-48b3-42e6-acef-317417870ae5.png|
|terrastation.network|https://urlscan.io/result/e9936135-136f-4b4e-9f40-9db71dee9055/|

### CODE REVIEW

**Comment by ical10** [comment]
WIP, creating screenshots and urlscans

**Comment by ical10** [comment]
ready for review!

File: all.json [hunk 1]
```diff
@@ -1,6 +1,26 @@
 {
 	"allow": [],
 	"deny": [
+		"all-in-oneconnect.com",
+		"syncnodelist.com",
+		"secured.all-in-oneconnect.com",
```
Comment (jacogr) [inline all.json:6]:
```suggestion
		"secured.all-in-oneconnect.com",
		"all-in-oneconnect.com",
```

Top-level block.

File: all.json [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+		"coin-claim.com",
+		"colablond.cc",
+		"collaplanb.cc",
+		"collablawd.cc",
+		"ragavio.ro",
```
Comment (jacogr) [inline all.json:11]: This name looks less problematic, cannot quite verify since the site itself is down.

**Review by jacogr — APPROVED** [review]

**Pull request merged**